### ISSUE TITLE
SDK version reporting

### ISSUE THREAD

**Issue by rmckeel** [balderdashy/sails.io.js#70]
The SDK_INFO version from sails.io.js is 0.11.0, but package.json and bower.json state differently (0.11.7).  Is this an oversight or expected?

Thanks!

  Ryan

**Comment by joshuamarquez** [comment]
same problem here

**Comment by sgress454** [comment]
As you can see in [the code](https://github.com/balderdashy/sails.io.js/blob/8d4fff63fcaead5f536bbed8ff4a0e3a1037247a/sails.io.js#L55), ideally we'd like to auto-generate that version # from the package.json.  If anyone wants to do a PR that will do this, we'd be happy to merge (it would involve putting some sort of template tag in sails.io.js and then adding something to the Grunt task to replace the tag when compiling the dist version).  Otherwise we'll get to it when we get to it, because at the moment all the backend cares about is whether the SDK is pre v0.11.x or not.

**Comment by rmckeel** [comment]
Hi @sgress454, thanks for the response!  No worries here now that we understand how to mute 'consolog' messages with 'sails.io.js#production' or using the minified version.  I won't be able to do that PR, so feel free to close this out if you'd like (or wait for someone else to do it).  Thanks!  Sails truly rocks.

  Ryan

**Comment by sailsbot** [comment]
Thanks for posting, @rmckeel.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
